PhotoLog_End/src/main/java/com/example/photolog_end/PermissionManager.kt

```diff
@@ -31,12 +32,18 @@ import kotlinx.coroutines.flow.asStateFlow
 
 class PermissionManager(private val context: Context) {
     companion object {
-        val REQUIRED_PERMISSIONS = arrayOf(
+        val REQUIRED_PERMISSIONS_PRE_T = arrayOf(
             READ_EXTERNAL_STORAGE,
             CAMERA,
             ACCESS_FINE_LOCATION,
             ACCESS_COARSE_LOCATION
         )
+        val REQUIRED_PERMISSIONS_POST_T = arrayOf(
+            Manifest.permission.READ_MEDIA_IMAGES,
```

Comment:
Nit: but since you add the static import for the others you might also do it for this one

Reply:
addressed, thanks Marcel